packages/server/graphql/private/typeDefs/_legacy.graphql

```diff
@@ -935,7 +935,7 @@ type JiraIssueEdge {
   The item at the end of the edge
   """
   node: JiraIssue!
-  cursor: DateTime
```

Comment:
This connection wasn't truly implemented previously, since our GQL API only accepted a `first` arg.

Using a `DateTime` cursor here makes things confusing when paginating through items in descending date order, since we'd be querying for everything before the `after` date. This also goes against [relay best-practices](https://relay.dev/graphql/connections.htm#sec-Cursor) that specify that cursors should be opaque to clients.

Using a normal `String` cursor also allows us to forward the `first` + `before` GQL connection args directly to the Atlassian API, eliminating complexity around converting our GQL pagination to the Atlassian API's pagination.

This also increases consistency with how the [Jira Server GQL API](https://github.com/ParabolInc/parabol/blob/2b9a5c9d7f0b7eb0e800264bff999e17a735c18f/packages/server/graphql/private/typeDefs/_legacy.graphql#L1035-L1061) is implemented